src/wallet/wallet.h

```diff
@@ -1541,7 +1559,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     std::optional<ZTXOSelector> ZTXOSelectorForAddress(
             const libzcash::PaymentAddress& addr,
             bool requireSpendingKey,
-            bool allowAddressLinkability) const;
+            bool requireTransparentCoinbase,
+            const TransactionStrategy& strategy) const;
```

Comment:
It took me a while to notice that the boolean argument had its interpretation altered, due to it looking at callsites like only a `TransactionStrategy` argument was being added. Ironically, if `requireTransparentCoinbase` were being inserted before `allowAddressLinkability`, I would have found it _easier_ to notice as I'd have been paying more attention to which boolean corresponded to which argument. In any case, if `requireTransparentCoinbase` is going to be left as an argument but `strategy` gets reverted to `allowAddressLinkability`, then put `requireTransparentCoinbase` last so the existing positional argument meanings are not changed.